frontend/src/app/app-routing.module.ts

```diff
@@ -10,33 +10,33 @@ const routes: Routes = [
         path: '',
         loadChildren: () => import('./modules/main/main.module').then((m) => m.MainModule),
         canMatch: [AuthorizedGuard],
+        canActivate: [HeaderGuard],
     },
     {
         path: '',
         loadChildren: () => import('./modules/landing/landing.module').then((m) => m.LandingModule),
-        canActivate: [HideHeaderGuard],
     },
     {
         path: 'auth',
         loadChildren: () => import('./modules/login/log-in.module').then((m) => m.LogInModule),
-        canActivate: [UnAuthorizedGuard, HideHeaderGuard],
+        canActivate: [UnAuthorizedGuard],
```

Comment:
maybe it is easier to hide the header when we do not need it?

Reply:
I saw the problem, that header downloads before page itself, and after that hides. And Hide Header guard couldn't solve this

https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/93576084/5a302168-72cf-4742-a4e4-2d9caab0d205